airflow/providers/github/__init__.py

```diff
@@ -39,5 +39,5 @@
     "2.5.0"
 ):
     raise RuntimeError(
-        f"The package `apache-airflow-providers-github:{__version__}` requires Apache Airflow 2.5.0+"
```

Comment:
Latest template applied here as well (cc: @eladkal - this the case whe you can use `--reapply-templates-only`

Changes in docs were done with:

```
breeze release-management prepare-provider-documentation github --reapply-templates-only
```